pkg/ingestor/parser/cyclonedx/parser_cyclonedx.go

```diff
@@ -448,33 +461,186 @@ func (c *cyclonedxParser) getAffectedPackages(ctx context.Context, vulnInput *mo
 
 	var viList []assembler.VexIngest
 	for _, affect := range *affectsObj.Range {
-		// TODO: Handle package range versions (see - https://github.com/CycloneDX/bom-examples/blob/master/VEX/CISA-Use-Cases/Case-8/vex.json#L42)
```

Comment:
@SantiagoTorres @arorasoham9 something to check against the identifier model of how to express such version ranges for CVEs and VEX.